### PR TITLE
Create an rss feed for the latest emails

### PR DESCRIPTION
**Pull request by duncan3dc**
This pr provides an endpoint (https://externals.io/rss) for an RSS feed of emails.

I currently use [the php.net rss feed](http://news.php.net/group.php?group=php.internals&format=rss) to create notifications for any new internals posts. And then link to news.php.net to display them, however I'd rather link to `externals.io`.

I tried to follow the conventions of the project, but let me know if there's anything I've done wrong

### CODE REVIEW

**Comment by mnapoli** [comment]
Thank you for the pull request, that's a really nice addition!

I'm working on #3 at the moment and I'm afraid it might change a bit of things. If you don't mind I'll wait till the migration is done to merge this.

**Comment by duncan3dc** [comment]
Sure thing, I'll keep an eye out for that being merged and then rebase this and resolve any issues

**Comment by mnapoli** [comment]
By the way would you want a RSS feed of the latest emails? Or latest threads? Maybe both?

**Comment by duncan3dc** [comment]
I'm only interested in emails, but I suppose there would be some use cases for threads too, shall I create `/rss/threads` and `/rss/emails`?

**Comment by mnapoli** [comment]
I was asking because I'm trying to merge threads and emails together (a thread is just an email without parent). Let's see if it works out and then we'll only need RSS for emails.

**Comment by mnapoli** [comment]
Thanks I had not seen that you rebased your commit!

File: src/RssBuilder.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        return $this->dom->saveXML();
+    }
+
+
+    private function addTextNode($name, $value, \DomElement $parent)
+    {
+        $element = $this->dom->createElement($name);
+
+        $value = utf8_encode($value);
```
Comment (staabm) [inline src/RssBuilder.php]: Why do we need this? Where do the iso8859 data come from?

Comment (mnapoli) [inline src/RssBuilder.php]: Indeed it should all be UTF8, I've removed this.

**Pull request merged**